R/qenv-eval_code.R

```diff
@@ -75,14 +75,13 @@ setMethod("eval_code", signature = c("qenv", "expression"), function(object, cod
 #' @rdname eval_code
 #' @export
 setMethod("eval_code", signature = c("qenv", "language"), function(object, code) {
-  code_char <- as.expression(code)
-  eval_code(object, code_char)
+  eval_code(object, code = format_expression(as.expression(code)))
 })
 
 #' @rdname eval_code
 #' @export
-setMethod("eval_code", signature = c("qenv", "character"), function(object, code) {
-  eval_code(object, code = parse(text = code, keep.source = FALSE))
+setMethod("eval_code", signature = c("qenv", "expression"), function(object, code) {
+  eval_code(object, code = format_expression(code))
```

Comment:
`format_expression` calls `styler::style_text` which proved it is slow several times. We shouldn't add anything to the `eval_code` which slows down evaluation.

 
```suggestion
  eval_code(object, code = code)
```